src/pages/set/[id].tsx

```diff
@@ -86,15 +112,18 @@ const Set = ({ tempIdx }: number | undefined) => {
               <button
                 onClick={() => setCurIndex(curIndex - 1)}
                 disabled={curIndex === 0}
-                className="flex flex-row items-center justify-center p-8"
+                className={`flex flex-row items-center justify-center p-8 ${curIndex === 0 ? "invisible" : "visible"}`}
               >
                 <Icon as={ChevronLeftIcon} />
                 Back
               </button>
               <button
                 onClick={() => setCurIndex(curIndex + 1)}
-                disabled={curIndex === cards.length - 1}
-                className="flex flex-row items-center justify-center p-8"
+                // If the user is at the end or if the user has not attempted to answer the question, disable the next button
+                disabled={
+                  curIndex === cards.length - 1 || maxIndex === curIndex
+                }
+                className={`flex flex-row items-center justify-center p-8 ${curIndex === cards.length - 1 || maxIndex === curIndex ? "invisible" : "visible"}`}
```

Comment:
Right now, I'm also hiding the button and disabling it. I think the disabled prop is redundant potentially since I changed the behavior of showing these buttons